tests/model_fields/storage.py

```diff
@@ -0,0 +1,10 @@
+from django.core.files.storage.filesystem import FileSystemStorage
+
+
+class NoReadException(Exception):
+    pass
+
+
+class NoReadFileSystemStorage(FileSystemStorage):
+    def open(self, *args, **kwargs):
+        raise NoReadException("This storage class does not support reading.")
```

Comment:
It’s for tests, we can use `AssertionError`, no need for the extra noise creating a custom exception class.

```suggestion
class NoReadFileSystemStorage(FileSystemStorage):
    def open(self, *args, **kwargs):
        raise AssertionError("This storage class does not support reading.")
```